tests/e2e/default/more-options-menu/offline-user/all-permissions.wdio-spec.js

```diff
@@ -39,15 +39,14 @@ const patient = personFactory.build({
   _id: uuid(),
   parent: { _id: clinic._id, parent: { _id: health_center._id, parent: { _id: district_hospital._id }}}
 });
-const xmlReport = reportFactory.build({ form: 'home_visit', content_type: 'xml' }, { patient, submitter: contact });
+const xmlReport = reportFactory.report().build(
+  { form: 'home_visit', content_type: 'xml' },
+  { patient, submitter: contact }
+);
```

Comment:
This is another thing that is really minor, but has tripped me up a lot in the past (and if we ever get some linting around this could cause issues).  Technically our coding style for [method chaining](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/#method-chaining) says that when chaining methods, they all need to go on their own line.  So, something like this:

```suggestion
const xmlReport = reportFactory
  .report()
  .build(
    { form: 'home_visit', content_type: 'xml' },
    { patient, submitter: contact }
  );
```